.travis.yml

```diff
@@ -47,22 +47,21 @@ jobs:
       - PYTHON=3.7
       - NUMPY=1.16.2
       - PANDAS>=0.24.1
-      - UPSTREAM_SPARSE=1
       - *test_and_lint
       - *no_coverage
       - *no_optimize
       - *imports
-      - *array_function
 
-    - env: &py36_dev
+    - env: &py37_dev
+      if: type == cron OR commit_message =~ test-upstream   # Run on cron jobs or any builds where commit message contains "test-upstream"
```

Comment:
You need to move this line to the bottom of the list (back where the previous `if:` block was. This is not currently a valid yaml file, which is why travis is ignoring it.

Reply:
Oh, I see, thanks for pointing that out! I thought having it at the top would make it more obvious that this test isn't always running, but didn't realize it was breaking the yaml file.